### ISSUE TITLE
[Platform] Proof of Concept - Scalability test: Test if Cluster API is enabled on the platform and manages a node

### ISSUE THREAD

**Issue by lixuna** [cncf/cnf-conformance#328]
[Acceptance Criteria](https://github.com/cncf/cnf-conformance/issues/328#issuecomment-670202310)
---

### [Platform] Proof of Concept - Scalability test: Test if Cluster API is enabled

**Test Category**
- Scalability

**What is the Benefit to operators?**
-  Scalability decreases risk and effort for Operators maintaining the platform

**Short description of platform test:** 
- the test suite will check if Cluster API is enabled and is managing a node

**Why is this test valuable?**
- Testing if Cluster API installed is a pre-requisite to using Cluster API in future platform tests
- Having Cluster API installed indicates that the platform could be using Cluster API to scale the cluster automatically

**Out of Scope**
- Updating the CNF Testbed example platform to add Cluster API
   -  a new ticket on CNF Testbed project would be needed
- Testing that Cluster API works for scaling
- Autoscaling using Cluster API

**Proof of Concept Tasks:**

Tasks:
- [x] Add a quick overview of the type of test you are researching/creating a POC for
- [x] Investigate potential tools for implementation and document findings, https://hackmd.io/new  (add link to ticket)
- [x] Select a tool to use, minimal/least effort, and add selection to ticket
- [x] Add new POC test code
- [x] Add comment suggesting updates as needed for:
  - [ ] the [test categories markdown](https://github.com/cncf/cnf-conformance/blob/master/TEST-CATEGORIES.md)
  - [ ] the [pseudo code markdown](https://github.com/cncf/cnf-conformance/blob/master/PSEUDO-CODE.md)
  - [ ] slide content updates, LINK_TO_UPDATES
  - [ ] the [README](https://github.com/cncf/cnf-conformance/blob/master/README.md)
- [x] Tag 1 or more people to peer review

### QA tasks

Dev Review: 
- [ ] walk through A/C
- [ ] do you get the expected result?
- [ ] if yes, 
   - [ ] move to `Needs Peer Review` column  
   - [ ] create Pull Request and follow check list
   - [ ] Assign 1 or more people for peer review
- [ ] if no, document what additional tasks will be needed

Peer review:
- [ ] walk through A/C
- [ ] do you get the expected result?
- [ ] if yes, 
   - [ ] move to `Reviewer Approved` column
   - [ ] Approve pull request
- [ ] if no, 
   - [ ] document what did not go as expected, including error messages and screenshots (if possible)
   - [ ] Add comment to pull request
   - [ ] request changes to pull request

**Comment by lixuna** [comment]
DRAFT Acceptance Criteria - based on https://github.com/cncf/cnf-conformance/issues/320#issuecomment-665803813
---

### Acceptance Criteria for peer review:
--
### Documentation is updated:
- [ ]  Documentation lists new test name
- [ ]  Test Categories md is updated
- [ ]  USAGE md is updated
- [ ]  Installation instructions is updated

###  Common steps
- [ ]  Find or create a K8s cluster to use
   - Talk with team for access to an existing cluster
   - [Kind K8s clusters](https://github.com/cncf/cnf-conformance/blob/master/KIND-INSTALL.md) will work as well
- [ ]  Set the KUBECONFIG environment variable for accessing the target K8s cluster
- [ ]   As a developer, Pull down latest version of CNF conformance test code from {FEATURE} branch
     - [ ]  Check that {FEATURE} Branch passes all A/C 
     - [ ]  Check that {FEATURE} Branch passes tests in Travis CI
- [ ]   As a peer reviewer, check that PR passes Travis CI tests
     - [ ]  if yes, merge PR to master
     - [ ]  if no, make a comment and request that developer make changes to pass tests 
- [ ]   As a peer reviewer, Pull down latest version of CNF conformance test code from `master` branch
- [ ]   Run `shards install`

### Spec test should pass
- [ ]   Run `crystal spec --tag platform`
- [ ]   I expect to see PASSED for the tests


### Update Prerequistite checker
- N/A

### Update example CNFs cnf-conformance.yml, if needed
- [ ] continue, If there is an update to cnf-conformance.yml
- [ ] for each CNF Example
- [ ] Update the example CNF's cnf-conformance.yml to include the new changes
- [ ] Update the example CNF's README if needed
- [ ] Test the example CNF following it's README

### Update yml validator, if needed
- N/A

### When using the test `cnf-conformance chaos_node_failure` command should PASS test successfully:
- [ ]  Run `./cnf-conformance setup`
- [ ]  Run  `./cnf-conformance sample_coredns_setup`
- [ ]  Run  `./cnf-conformance chaos_node_failure`
- [ ]  I expect to see the output and results from the test
- [ ]  I expect to see PASSED for the test. Example:
         - [ ] "✔️ PASSED:  Cluster recovered after node failure [EMOJI]"

### When using the test `crystal spec --tag `chaos_node_failure` command the sample "good" CNF should PASS and the "bad example CNF"  should FAIL test:
- [ ]   Run `crystal spec --tag volume_hostpath_not_found`
- [ ]   I expect to see the output and results from the test
- [ ]   I expect to see "✔️ PASSED: Cluster recovered after node failure [EMOJI]"
- [ ]   I expect to see "❌ FAILED: Cluster did not recover after node failure [EMOJI]"

### I would expect the CNF to pass when I run the entire test category suite (`resilience`)
- [ ]  Follow common steps above
- [ ]  Run  `./cnf-conformance sample_coredns_setup`
- [ ]  Run  `./cnf-conformance resilience`
- [ ]   I expect to see the output and results from all tests in the test category
- [ ]  I expect to see PASSED for the test and test category 

### I would expect the CNF to pass when I run the entire test suite for platform tests: (`platform`)
- [ ]   Follow common steps above
- [ ]   Run  `./cnf-conformance sample_coredns_setup`
- [ ]   Run  `./cnf-conformance platfrom`
- [ ]   I expect to see the output from the test
- [ ]   I expect to see PASSED for the test

### Screenshots of new test and results in the comments
- [ ]  New test should be added to the points file
- [ ]  New test score should be added to the final score
- [ ] I can see a screenshot of the new test in the comments of this ticket

**Comment by taylor** [comment]
8pts

**Comment by lixuna** [comment]
Level of effort in pts: 0,1,2,3,5,8,13,21 - 8 pts
---

@wavell 5
@williscool 5
@denverwilliams 8 (cluster api learning curve)
@agentpoyo 8
@taylor 8
@lixuna 8

**Comment by taylor** [comment]
reference:

https://github.com/chuckha/capi-dev
> Cluster API Development Environment
> This repo houses some tools to make running Cluster API v1alpha2 a little bit easier.

**Comment by williscool** [comment]
Googled cluster emoji and this was first result

https://emojipedia.org/sparkles/

**Comment by williscool** [comment]
stuck here for the day


working on either docker


https://cluster-api.sigs.k8s.io/clusterctl/developers.html#create-a-clusterctl-settingsjson-file


 or packet 

https://cluster-api.sigs.k8s.io/user/quick-start.html#initialization-for-common-providers


setup so that we can create / orchestrate a cluster with `clusterctl` and then see if there are labels on the nodes and the management cluster communicates with the workload cluster and test for that

**Comment by williscool** [comment]
Notes on stuff so far for setting up management cluster... hopefully 1 step away

how to cluster api
---

based on cluster-api/docs/book/src/developer/guide.md


## prereqs

- existing cluster https://cluster-api.sigs.k8s.io/user/quick-start.html#install-andor-configure-a-kubernetes-cluster
- `clusterclt` https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl
- a registry. you can easily setup one with https://github.com/tilt-dev/kind-local/
  - `KIND_CLUSTER_NAME='kind-registry' ./kind-with-registry.sh` 



```
cat > kind-cluster-with-extramounts-with-registry-access.yaml <<EOF
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
containerdConfigPatches:
- | # https://github.com/tilt-dev/kind-local/blob/master/kind-with-registry.sh
  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:5000"]
    endpoint = ["http://kind-registry:5000"]
nodes:
  - role: control-plane
    extraMounts:
      - hostPath: /var/run/docker.sock
        containerPath: /var/run/docker.sock
EOF

kind create cluster --name kind-cluster-api-manager --config kind-cluster-with-extramounts-with-registry-access.yaml

kind get kubeconfig --name kind-cluster-api-manager > kind-cluster-api-manager-kubeconfig.yaml
export KUBECONFIG=$(pwd)/kind-cluster-api-manager-kubeconfig.yaml


cat > kind-local-cluster-config-map.yaml <<EOF
# run the script from https://github.com/tilt-dev/kind-local first!
apiVersion: v1
kind: ConfigMap
metadata:
  name: local-registry-hosting
  namespace: kube-public
data:
  localRegistryHosting.v1: |
    host: "localhost:5000"
    help: "https://kind.sigs.k8s.io/docs/user/local-registry/"
EOF

kubectl apply -f kind-local-cluster-config-map.yaml
```





## build docker containers

``` bash
git clone https://github.com/kubernetes-sigs/cluster-api

## parts of the cluster api dev setup scripts try, and generally fail lol, to install cert-manager just install ahead of time to avoid the headaches
helm repo add jetstack https://charts.jetstack.io
helm install --create-namespace \
  cert-manager jetstack/cert-manager \
  --namespace cert-manager \
  --version v0.16.1 \
  --set installCRDs=true


cd < parent_dir i.e. ../ >

# i know this says aws even though you won't be using it
# the tilfile python script breaks if this isn't there trust me lol
git clone git@github.com:kubernetes-sigs/cluster-api-provider-aws.git

cd ../cluster-api



cat > tilt-settings.json <<EOF
{
    "default_registry": "localhost:5000",
    "provider_repos": [],
    "enable_providers": [
        "docker",
        "kubeadm-bootstrap",
        "kubeadm-control-plane"
    ]
}
EOF
```



patch `Tiltfile`


``` diff
william@Will-LAPTOP-L47SIB7:/mnt/c/Users/william/UNSYNCED-symlinked-to-linux/cluster-api$ git diff
diff --git a/Tiltfile b/Tiltfile
index ef4a26158..5fde5a684 100644
--- a/Tiltfile
+++ b/Tiltfile
@@ -161,7 +161,10 @@ metadata:
   namespace: cert-manager-test
 spec:
   dnsNames:
-    - example.com
+    - *.svc.cluster.local
+    - *.docker.internal
+    - localhost
+    - 127.0.0.1
   secretName: selfsigned-cert-tls
   issuerRef:
     name: test-selfsigned
@@ -176,7 +179,7 @@ def enable_provider(name):
     p = providers.get(name)

     context = p.get("context")
-    go_main = p.get("go_main")
+    go_main = p.get("go_main", "main.go")

     # Prefix each live reload dependency with context. For example, for if the context is
     # test/infra/docker and main.go is listed as a dep, the result is test/infra/docker/main.go. This adjustment is
@@ -263,9 +266,9 @@ def deploy_cert_manager():
     local("kubectl wait --for=condition=Available --timeout=300s -n cert-manager deployment/cert-manager-webhook")

     # 2. create a test certificate
-    local("cat << EOF | kubectl apply -f - " + cert_manager_test_resources + "EOF")
-    local("kubectl wait --for=condition=Ready --timeout=300s -n cert-manager-test certificate/selfsigned-cert ")
-    local("cat << EOF | kubectl delete -f - " + cert_manager_test_resources + "EOF")
+    # local("cat << EOF | kubectl apply -f - " + cert_manager_test_resources + "EOF")
+    # local("kubectl wait --for=condition=Ready --timeout=300s -n cert-manager-test certificate/selfsigned-cert ")
+    # local("cat << EOF | kubectl delete -f - " + cert_manager_test_resources + "EOF")

```



install tilt 

```
brew install tilt
```

run tilt

```
tilt up
```



wait for it to build all of the docker containers they are used to build control plane nodes and such



## Install dev docker infrastructure provider so `clusterctl` can use it

based on https://networkop.co.uk/post/2020-05-cluster-api-intro/



``` bash
cat > clusterctl-settings.json <<EOF
{
  "providers": ["cluster-api","bootstrap-kubeadm","control-plane-kubeadm", "infrastructure-docker"]
}
EOF

./cmd/clusterctl/hack/create-local-repository.py
```



The `create-local-repository.py` script  will give you a `clusterctl` command to run ...  **but don't run it yet!**



``` bash
cat > ~/.cluster-api/clusterctl.yaml <<EOF
providers:
  - name: docker
    url: $HOME/.cluster-api/dev-repository/infrastructure-docker/v0.3.8/infrastructure-components.yaml
    type: InfrastructureProvider
EOF

# then 
# why doesn't the script install this for you? beats me lol. what does it do? also beats me
cp test/e2e/data/infrastructure-docker/metadata.yaml ~/.cluster-api/dev-repository/infrastructure-docker/
```

fix up the yaml for the infra provider

``` bash

cd ~/.cluster-api/dev-repository

# check in all of the subdirectories and make sure each .yaml file creates namespaces with the capi- prefix
# re: https://github.com/kubernetes-sigs/cluster-api/issues/3426#issuecomment-679966615
# i.e. in 

vim infrastructure-docker/v0.3.8/infrastructure-components.yaml

# change name: system to name: capi-system

# then inisd of 

vim infrastructure-docker/v0.3.8/infrastructure-components.yaml

# replace the image image: gcr.io/k8s-staging-cluster-api/capd-manager:dev
# with the local version that was builit by tilt in teh first step for you i.e

docker images localhost:5000/gcr.io_k8s-staging-cluster-api_cluster-api-controller

localhost:5000/gcr.io_k8s-staging-cluster-api_cluster-api-controller
```



now you can run the `clusterctl` command i.e.

```
# if you have a cert-manager failure just try again	
clusterctl init  --core cluster-api:v0.3.8  --bootstrap kubeadm:v0.3.8    --control-plane kubeadm:v0.3.8    --infrastructure docker:v0.3.8    --config ~/.cluster-api/dev-repository/config.yaml
```



verify everything is ready  (stuck here 9-15-2020)



```
kubectl get deploy -A | grep cap
kubectl -n capi-system describe pod capd-controller-manager<TAB>-<TAB>
```





## Build a workload cluster

(not yet tested)

```
DOCKER_POD_CIDRS="192.168.0.0/16" \
DOCKER_SERVICE_CIDRS="10.128.0.0/12" \
DOCKER_SERVICE_DOMAIN="cluster.local" \
clusterctl config cluster capd --kubernetes-version v1.17.5 \
--from ./test/e2e/data/infrastructure-docker/cluster-template.yaml \
--target-namespace default \
--control-plane-machine-count=1 \
--worker-machine-count=1 \
> capd.yaml
```

**Comment by williscool** [comment]
how to cluster api
---

based on cluster-api/docs/book/src/developer/guide.md


## prereqs

- existing cluster https://cluster-api.sigs.k8s.io/user/quick-start.html#install-andor-configure-a-kubernetes-cluster
- `clusterclt` https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl
- a registry. you can easily setup one with https://github.com/tilt-dev/kind-local/
  - `KIND_CLUSTER_NAME='kind-registry' ./kind-with-registry.sh` 



```
# https://github.com/kubernetes-sigs/cluster-api/issues/3013#issuecomment-668154466
export KIND_EXPERIMENTAL_DOCKER_NETWORK=bridge

cat > kind-cluster-with-extramounts-with-registry-access.yaml <<EOF
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
containerdConfigPatches:
- | # https://github.com/tilt-dev/kind-local/blob/master/kind-with-registry.sh
  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:5000"]
    endpoint = ["http://kind-registry:5000"]
nodes:
  - role: control-plane
    extraMounts:
      - hostPath: /var/run/docker.sock
        containerPath: /var/run/docker.sock
EOF

kind create cluster --name kind-cluster-api-manager --config kind-cluster-with-extramounts-with-registry-access.yaml

kind get kubeconfig --name kind-cluster-api-manager > kind-cluster-api-manager-kubeconfig.yaml
export KUBECONFIG=$(pwd)/kind-cluster-api-manager-kubeconfig.yaml


cat > kind-local-cluster-config-map.yaml <<EOF
# run the script from https://github.com/tilt-dev/kind-local first!
apiVersion: v1
kind: ConfigMap
metadata:
  name: local-registry-hosting
  namespace: kube-public
data:
  localRegistryHosting.v1: |
    host: "localhost:5000"
    help: "https://kind.sigs.k8s.io/docs/user/local-registry/"
EOF

kubectl apply -f kind-local-cluster-config-map.yaml
```





## build docker containers

``` bash
git clone https://github.com/kubernetes-sigs/cluster-api --branch v0.3.9

## parts of the cluster api dev setup scripts try, and generally fail lol, to install cert-manager just install ahead of time to avoid the headaches
helm repo add jetstack https://charts.jetstack.io
helm install --create-namespace \
  cert-manager jetstack/cert-manager \
  --namespace cert-manager \
  --version v0.16.1 \
  --set installCRDs=true


cd < parent_dir i.e. ../ >

# i know this says aws even though you won't be using it
# the tilfile python script breaks if this isn't there trust me lol
git clone git@github.com:kubernetes-sigs/cluster-api-provider-aws.git

cd ../cluster-api



cat > tilt-settings.json <<EOF
{
    "default_registry": "localhost:5000",
    "provider_repos": [],
    "enable_providers": [
        "docker",
        "kubeadm-bootstrap",
        "kubeadm-control-plane"
    ]
}
EOF
```



patch `Tiltfile`


``` diff
william@Will-LAPTOP-L47SIB7:/mnt/c/Users/william/UNSYNCED-symlinked-to-linux/cluster-api$ git diff
diff --git a/Tiltfile b/Tiltfile
index ef4a26158..5fde5a684 100644
--- a/Tiltfile
+++ b/Tiltfile
@@ -161,7 +161,10 @@ metadata:
   namespace: cert-manager-test
 spec:
   dnsNames:
-    - example.com
+    - *.svc.cluster.local
+    - *.docker.internal
+    - localhost
+    - 127.0.0.1
   secretName: selfsigned-cert-tls
   issuerRef:
     name: test-selfsigned
@@ -176,7 +179,7 @@ def enable_provider(name):
     p = providers.get(name)

     context = p.get("context")
-    go_main = p.get("go_main")
+    go_main = p.get("go_main", "main.go")

     # Prefix each live reload dependency with context. For example, for if the context is
     # test/infra/docker and main.go is listed as a dep, the result is test/infra/docker/main.go. This adjustment is
@@ -263,9 +266,9 @@ def deploy_cert_manager():
     local("kubectl wait --for=condition=Available --timeout=300s -n cert-manager deployment/cert-manager-webhook")

     # 2. create a test certificate
-    local("cat << EOF | kubectl apply -f - " + cert_manager_test_resources + "EOF")
-    local("kubectl wait --for=condition=Ready --timeout=300s -n cert-manager-test certificate/selfsigned-cert ")
-    local("cat << EOF | kubectl delete -f - " + cert_manager_test_resources + "EOF")
+    # local("cat << EOF | kubectl apply -f - " + cert_manager_test_resources + "EOF")
+    # local("kubectl wait --for=condition=Ready --timeout=300s -n cert-manager-test certificate/selfsigned-cert ")
+    # local("cat << EOF | kubectl delete -f - " + cert_manager_test_resources + "EOF")

```



install tilt 

```
brew install tilt
```

run tilt

```
tilt up
```



wait for it to build all of the docker containers they are used to build control plane nodes and such



push the docker images to ur local registry

```
docker tag localhost:5000/gcr.io_k8s-staging-cluster-api_cluster-api-controller:tilt-59e7996872fb6708 localhost:5000/gcr.io_k8s-staging-cluster-api_cluster-api-controller

docker push localhost:5000/gcr.io_k8s-staging-cluster-api_cluster-api-controller
```



## Install dev docker infrastructure provider so `clusterctl` can use it

based on https://networkop.co.uk/post/2020-05-cluster-api-intro/



``` bash
cat > clusterctl-settings.json <<EOF
{
  "providers": ["cluster-api","bootstrap-kubeadm","control-plane-kubeadm", "infrastructure-docker"]
}
EOF

./cmd/clusterctl/hack/local-overrides.py
```



The `local-overrides.py` script  will give you a `clusterctl` command to run ...  **but don't run it yet!**



``` bash
cat > ~/.cluster-api/clusterctl.yaml <<EOF
providers:
  - name: docker
    url: $HOME/.cluster-api/dev-repository/infrastructure-docker/v0.3.0/infrastructure-components.yaml
    type: InfrastructureProvider
EOF
```

fix up the yaml for the infra provider

``` bash

cd ~/.cluster-api/overrides

# check in all of the subdirectories and make sure each .yaml file creates Namespace resources with the capi- prefix
# re: https://github.com/kubernetes-sigs/cluster-api/issues/3426#issuecomment-679966615
# i.e. in 

vim infrastructure-docker/v0.3.0/infrastructure-components.yaml

# change name: system to name: capi-system

# inside of 

 vim cluster-api/v0.3.0/core-components.yaml

# change name: webhook-system to name: capi-webhook-system

# then inside of 

vim infrastructure-docker/v0.3.0/infrastructure-components.yaml

# change name: webhook-system to name: capd-webhook-system

# replace the image image: gcr.io/k8s-staging-cluster-api/capd-manager:dev
# with the local version that was builit by tilt in teh first step for you i.e

docker images localhost:5000/gcr.io_k8s-staging-cluster-api_cluster-api-controller

localhost:5000/gcr.io_k8s-staging-cluster-api_cluster-api-controller
```



now you can run the `clusterctl` command i.e.

```
# if you have a cert-manager failure just try again	
clusterctl init  --core cluster-api:v0.3.0  --bootstrap kubeadm:v0.3.0    --control-plane kubeadm:v0.3.0    --infrastructure docker:v0.3.0 
```



verify everything is ready 


```
kubectl get deploy -A | grep cap
kubectl -n capi-system describe pod capd-controller-manager<TAB>-<TAB>
```



## Build a workload cluster

(9-15-2020: stuck here: its looking for a `CNI_RESOURCES` variable i don;t know how to set)

```
CNI_RESOURCES="./data/cni/kindnet/kindnet.yaml" \
DOCKER_POD_CIDRS="127.0.0.240/28" \
DOCKER_SERVICE_CIDRS="10.128.0.0/12" \
DOCKER_SERVICE_DOMAIN="cluster.local" \
clusterctl config cluster capd --kubernetes-version v1.17.5 \
--from ./test/e2e/data/infrastructure-docker/cluster-template.yaml \
--target-namespace default \
--control-plane-machine-count=1 \
--worker-machine-count=1 \
> capd.yaml
```



```
kubectl apply -f capd.yaml
```

**Comment by williscool** [comment]
I believe the issue is related to CNI setup

**Comment by williscool** [comment]
![image](https://user-images.githubusercontent.com/394931/94083418-ecafad00-fdb7-11ea-914b-5e0500c1126d.png)


screenshot

**Comment by denverwilliams** [comment]
@williscool @taylor @lixuna 

I peer reviewed the test and it looks good, but i noticed there a re no spec tests.

The main caveat with having a positive spec test will be the need to have cluster-api setup on the cluster we are running spec against. So i played around with creating the minimum required CRDs and API objects for the test to pass without having to truly setup cluster-api. I pushed up a file with the commands [here](https://github.com/cncf/cnf-conformance/commit/0fce5aab988cb7134bca7f259b0d933dfde50c99)

You should hopefully be able to write a spec test around it fairly quickly.

**Comment by williscool** [comment]
@denverwilliams  spec test

https://github.com/cncf/cnf-conformance/pull/417

unfortunately the script isn't quite all the way there 

getting an error when applying the capd file

![image](https://user-images.githubusercontent.com/394931/94779230-9bde1c80-0394-11eb-912e-d42015511508.png)

**Comment by lixuna** [comment]
@williscool what's remaining on this issue? Is it ready for peer review (if so please drag to next column)? Thanks!

**Comment by williscool** [comment]
per @denverwilliams  Working on 

waiting for the "crd resources to finish creating before running the kubectl apply."

in the spec test

**Comment by wavell** [comment]
![image](https://user-images.githubusercontent.com/29548/97226509-120e4d00-17a2-11eb-916d-bea9aa85ceee.png)

**Comment by wavell** [comment]
Cluster API now has a negative test (a spec that runs without installed cluster api first)

**Comment by lixuna** [comment]
@wavell  @denverwilliams this issue’s peer review is blocked by “clusterapi_enabled test has not been added to points.yml”

Given that this is a proof of concept, should we expect to see points for this test?

**Comment by nupejosh** [comment]
@lixuna Just spoke with @wavell about this. We do not need to see points for this test bc it is POC. I'll disregard those checkboxes in the A/C to get this ticket closed out.

**Comment by nupejosh** [comment]
![Screen Shot 2020-11-24 at 2 55 25 PM](https://user-images.githubusercontent.com/3164983/100150360-18f1b380-2e65-11eb-8c2e-3bebe0b5b1f6.png)
![Screen Shot 2020-11-24 at 2 55 40 PM](https://user-images.githubusercontent.com/3164983/100150376-20b15800-2e65-11eb-9375-5cb59a3a53e4.png)

**Comment by lixuna** [comment]
Thanks for confirming, @nupejosh. Closing issue
